packit/status.py

```diff
@@ -117,17 +116,20 @@ def get_builds(self,) -> Dict:
         """
         Get latest koji builds
         """
-        # https://github.com/fedora-infra/bodhi/issues/3058
-        from bodhi.client.bindings import BodhiClient
-
-        b = BodhiClient()
-        # { koji-target: "latest-build-nvr"}
-        builds_d = b.latest_builds(self.dg.package_config.downstream_package_name)
-        logger.debug(f"Latest Koji builds fetched (from Bodhi): {builds_d}.")
-        builds: Dict = {}
-        for tag, nvr in builds_d.items():
-            if tag.endswith("-updates-candidate"):
-                builds[tag[: -len("-updates-candidate")]] = nvr
+        session = ClientSession(baseurl="https://koji.fedoraproject.org/kojihub")
+        package_id = session.getPackageID(
+            self.dg.package_config.downstream_package_name
+        )
+        # We don't need whole history, get just recent 6 months
+        since = datetime.now() - timedelta(days=180)
```

Comment:
What about some constant or, ideally, some (CLI) option for that?

(Since it's not so important, we can create some good-first-issue for that afterwards.)

Reply:
We don't return the delta-length history of builds, only the latest build per branch, so this delta has no value for end-user.
Therefore it makes no sense to expose it to users.

I increased it to a year, which should be sufficient for anyone and added some more comments.